isis/appdata/import/PDS3/MroHirise.tpl

```diff
@@ -0,0 +1,99 @@
+    PrimaryMirrorMntTemperature     = {{TEMPERATURE_PARAMETERS.MRO_PRIMARY_MIRROR_MNT_TEMPERATURE.Value}}
+    PrimaryMirrorTemperature        = {{TEMPERATURE_PARAMETERS.MRO_PRIMARY_MIRROR_MNT_TEMPERATURE.Value}}
+    PrimaryMirrorBafTemperature     = {{TEMPERATURE_PARAMETERS.MRO_PRIMARY_MIRROR_BAF_TEMPERATURE.Value}}
+    MsTrussLeg0ATemperature         = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_0_A_TEMPERATURE.Value}}
+    MsTrussLeg0BTemperature         = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_0_B_TEMPERATURE.Value}}
+    MsTrussLeg120ATemperature       = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_120_A_TEMPERATURE.Value}}
+    MsTrussLeg120BTemperature       = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_120_B_TEMPERATURE.Value}}
+    MsTrussLeg240ATemperature       = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_240_A_TEMPERATURE.Value}}
+    MsTrussLeg240BTemperature       = {{TEMPERATURE_PARAMETERS.MRO_MS_TRUSS_LEG_240_B_TEMPERATURE.Value}}
+    BarrelBaffleTemperature         = {{TEMPERATURE_PARAMETERS.MRO_BARREL_BAFFLE_TEMPERATURE.Value}}
+    SunShadeTemperature             = {{TEMPERATURE_PARAMETERS.MRO_SUN_SHADE_TEMPERATURE.Value}}
+    SpiderLeg30Temperature          = {{TEMPERATURE_PARAMETERS.MRO_SPIDER_LEG_30_TEMPERATURE.Value}}
+    SpiderLeg150Temperature         = {{TEMPERATURE_PARAMETERS.MRO_SPIDER_LEG_150_TEMPERATURE.Value}}
+    SpiderLeg270Temperature         = {{TEMPERATURE_PARAMETERS.MRO_SPIDER_LEG_270_TEMPERATURE.Value}}
+    SecMirrorMtrRngTemperature      = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_MTR_RNG_TEMPERATURE.Value}}
+    SecMirrorTemperature            = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_TEMPERATURE.Value}}
+    SecMirrorBaffleTemperature      = {{TEMPERATURE_PARAMETERS.MRO_SEC_MIRROR_BAFFLE_TEMPERATURE.Value}}
+    IeaTemperature                  = {{TEMPERATURE_PARAMETERS.MRO_IEA_TEMPERATURE.Value}}
+    FocusMotorTemperature           = {{TEMPERATURE_PARAMETERS.MRO_FOCUS_MOTOR_TEMPERATURE.Value}}
+    IePwsBoardTemperature           = {{TEMPERATURE_PARAMETERS.MRO_IE_PWS_BOARD_TEMPERATURE.Value}}
+    CpmmPwsBoardTemperature         = {{TEMPERATURE_PARAMETERS.MRO_CPMM_PWS_BOARD_TEMPERATURE.Value}}
+    MechTlmBoardTemperature         = {{TEMPERATURE_PARAMETERS.MRO_MECH_TLM_BOARD_TEMPERATURE.Value}}
+    InstContBoardTemperature        = {{TEMPERATURE_PARAMETERS.MRO_INST_CONT_BOARD_TEMPERATURE.Value}}
+    DllLockedFlag                   = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_LOCKED_FLAG.Value, ", ")}})
+    DllResetCount                   = {{INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_RESET_COUNT.Value}}
+    DllLockedOnceFlag               = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_LOCKED_ONCE_FLAG.Value, ", ")}})
+    DllFrequenceCorrectCount        = {{INSTRUMENT_SETTING_PARAMETERS.MRO_DLL_FREQUENCY_CORRECT_COUNT.Value}}
+    ADCTimingSetting                = ({{ join(INSTRUMENT_SETTING_PARAMETERS.MRO_ADC_TIMING_SETTINGS.Value, ", ")}})
+  {% endblock %}
```

Comment:
There is an "UNLUTTED" key in the old test data. Is it supposed to be missing.

Reply:
I think it is coming from the applications. When it unluts the data it adds this key, so you don't have to add it yet. When we get to the DN's next year this will have to be addressed.